.gitignore

```diff
@@ -363,3 +363,5 @@ MigrationBackup/
 FodyWeavers.xsd
 /Warp.WebApp/appsettings.Local.json
 /data
+/Warp.WebApp/appsettings.json
+/Warp.WebApp/Properties/launchSettings.json
```

Comment:
We require these two files to build the app on a server. So please, bring them back.

If you need a change in `appsettings.json` you may add a `appsettings.Local.json` which is already ignored. All settings from `appsettings.Local.json` will override settings from `appsettings.json`.

As for `launchSettings.json`, you may add configurations to it as many as you want.